.pipelines/ci.yml

```diff
@@ -72,7 +72,7 @@ stages:
       displayName: Unpack ONNXRuntime package.
 
     - script: |
-        CPU_NUMBER=2 sh ./build.sh -DOCOS_ENABLE_CTEST=ON -DOCOS_ONNXRUNTIME_VERSION="$(ort.version)" -DONNXRUNTIME_PKG_DIR=$(Build.SourcesDirectory)/onnxruntime-linux-x64-$(ort.version)
+        CPU_NUMBER=8 sh ./build.sh -DOCOS_ENABLE_CTEST=ON -DOCOS_ONNXRUNTIME_VERSION="$(ort.version)" -DONNXRUNTIME_PKG_DIR=$(Build.SourcesDirectory)/onnxruntime-linux-x64-$(ort.version)
```

Comment:
The existing setting cpu_number=2 is due to memory constraint, since compiling OpenCV costs a lot of memory and the high concurrence compiling jobs will easily crash because of Out of memory. Let's try this number on the new machine pool